src/sql/base/common/locConstants.ts

```diff
@@ -38,6 +38,8 @@ export function extensionsWorkbenchServiceIncompatible(extension: string, extens
 export const fileActionsContributionNewQuery = localize('newQuery', "New Query");
 export const fileActionsContributionMiNewQuery = localize({ key: 'miNewQuery', comment: ['&& denotes a mnemonic'] }, "New &&Query");
 export const fileActionsContributionMiNewNotebook = localize({ key: 'miNewNotebook', comment: ['&& denotes a mnemonic'] }, "&&New Notebook");
+export const fileActionsContributionNewDeployment = localize('newDeployment', "New Deployment...");
```

Comment:
You should actually be using this - take a look at the new query examples (such as adding to the command palette) and make sure we have the same contributions covered.

Reply:
So we shouldn't be using the mnemonic version?

Reply:
The mnemonic version would be used in menus - so stuff like the file menu. Just look at how both types of the New Query contributions are used - that should serve as a good baseline to start with (there's ~3 different locations IIRC)

Reply:
I'm unsure of where this is coming from because I can't find it in ADS, but there's a command palette action registered for new deployment already.
![image](https://user-images.githubusercontent.com/87730006/222576720-2302f878-01f8-4452-a043-3bb93fc50c82.png)


Everything else looks good though.

Reply:
Ah, right. That's coming from https://github.com/Microsoft/azuredatastudio/blob/c024cc3c23138baaa4e71c2d49135fad08e876d8/extensions/resource-deployment/package.json#L48. You can delete `fileActionsContributionNewDeployment` then - it shouldn't be needed in core.